pics/chapter-swissair.tex

```diff
@@ -1,2 +1,2 @@
 Swiss International Air Lines AG (also known as Swiss), is the flag carrier company of Switzerland.
-The airline was formed following the failure of Swissair in 202.
\ No newline at end of file
+The airline was formed following the failure of Swissair in 2002.
```

Comment:
Since we are changing this line, maybe we could rephrase as "following the bankruptcy of Swissair in 2002"